drivers/video/vnc/vnc_fbdev.c

```diff
@@ -610,6 +612,22 @@ int up_fbinitialize(int display)
       return ret;
     }
 
+  /* Save the input callout function information in the session structure. */
+
+  session           = g_vnc_sessions[display];
+#ifdef CONFIG_VNCSERVER_TOUCH
+  ret = vnc_touch_register(CONFIG_VNCSERVER_TOUCH_DEVNAME, &session->touch);
+
+  if (ret < 0)
```

Comment:
sorry, look like the background thread is start at line vnc_start_server, so we need move before vnc_start_server